src/components/PlioQuestion.vue

```diff
@@ -7,8 +7,7 @@
         <div class="modal__body">
           <!-- The cross button to close the modal -->
           <div class="question_text_row">
-            <div class="question_text">
-              {{ plioQuestion.item.question.text }}
+            <div class="question_text" id="question" v-html="questionText">
```

Comment:
Instead of rendering the text as just a string (`{{ plioQuestion.item.question.text }}`), using a `v-html` directive will tell Vue to treat the text as HTML. 
The question information coming from the CMS is actually HTML content